legend-engine-core/legend-engine-core-pure/legend-engine-pure-code-compiled-core/src/main/resources/core/pure/router/store/routing.pure

```diff
@@ -34,7 +37,17 @@ import meta::pure::store::*;
 // =========================================================================================
 function meta::pure::router::store::routing::getRoutingStrategyFromMappingAndRuntime(mapping:Mapping[1], runtime:Runtime[1]):StoreMappingRoutingStrategy[1]
 {
-    getRoutingStrategyFromMappingAndRuntime($mapping, $runtime, []);
+  ^StoreMappingRoutingStrategy(
```

Comment:
Isn't this changing functionality? Previously it would have gone to function below to help with model joins based on runtime locality

Reply:
No, the function was unused before, I repurposed it to return just the strategy object. Both usages of this function are in the same file, the overloaded version of the function (with extensions param) is the one used externally. Have made this function private to avoid incorrect usage.